docs/UserGuide.md

```diff
@@ -35,100 +39,194 @@
+> * The format for date must be *day month year* where year is optional.<br>
+Eg: 10th Oct, 4 November, 11th August 2017. Not 12/12/12. <br>
+> * Time should be in the *12 hour format*.<br>
+Eg: 10am, 6pm, 7.30pm. Not 7:30pm, 2359.
+
+Examples:
+* `add "Do CS2103T tutorial" on 4 Oct 2016 from 10am to 11am every week`<br>
+Adds a recurring scheduled task �CS2103T tutorial� on the 4th October 2016 from 10am to 11am every week. 
+* `add "Watch Storks movie" on 10/10 from 1pm to 3pm`<br>
+Adds a scheduled task �Watch Storks movie� on 10th October 2016 from 1pm to 3pm. 
+* `add "CS2103 Project" by 20/11/2016 high`<br>
+Adds a scheduled task �CS2103 Project� of high priority on 20th November 2016. 
+* `add �CS2105 Tutorial� on Thursday at 11am till 12pm every week`<br>
+Adds a recurring scheduled task �CS2105 Tutorial� on Thursday from 11am to 12pm.
+* `add "Buy pizza" at 7pm`<br>
+Adds a scheduled task �Buy pizza� today at 7pm. 
+* `add "Charlie Puth concert� on 15th Oct 6pm"`<br>
+Adds a scheduled task �Charlie Puth concert� on 15th October at 6pm.
+* `add "Buy chocolate milk"`<br>
+Adds a todo task �Buy chocolate milk�.
 
-> Persons can have any number of tags (including 0)
+>Tasks with date and time specified will be added to WhatNow as a schedule task and will<br> be displayed under the heading "Schedule".<br> 
+> <img src="images/UI_Prototype/WhatNowUI-AddSchedule-Colour.png" width="600">
+
+>Tasks without date and time specified will be added to WhatNow as a todo task and will<br> be displayed under the heading "Todo Tasks".
+> <img src="images/UI_Prototype/WhatNowUI-AddTodo-Colour.png" width="600">
+
+
+#### Listing all tasks: `list`
+Shows a list of all tasks that match task type, date, time and priority requested by user.<br>
+Format: `list [date/time/priority] KEYWORD`
+
+> * Shows a list of all tasks that match the keyword for the task type, date, time and priority requested by user.
+
+Examples: 
+* `list`<br>
+Returns a list of all tasks. <br>
+<img src="images/UI_Prototype/WhatNowUI-List-Colour.png" width="600">
+* `list completed`<br>
+ Returns a list of all completed tasks. <br>
+* `list todo`<br>
+Returns a list of all todo tasks. <br>
+* `list priority medium`<br>
+Returns a list of all tasks that have a medium priority. <br>
+* `list time 6pm`<br>
+Returns a list of all tasks scheduled at 6pm. <br>
+* `list date 3 Nov`<br>
+Returns a list of all tasks scheduled for 3 November of the current year. <br>
+
+>If you want to view any of the Todo tasks, Scheduled tasks or Completed tasks only, then you only<br> need to type in �todo�, �schedule� or �completed� as the keyword after the command list. <br>
+> <img src="images/UI_Prototype/WhatNowUI-ListCompleted-Colour.png" width="600">
+
+
+#### Deleting a task : `delete`
+Deletes a task from WhatNow. Reversible.<br>
+Format: `delete todo/schedule/completed INDEX`
+
+> * Deletes the task at the specified `INDEX` from todo tasks, schedule or completed tasks as specified. <br>
```

Comment:
oh sorry I think we need to tell the user to list completed before deleting a completed task.